gradle.properties

```
@@ -0,0 +1,404 @@
+model.all.version=1.0.0
```

```suggestion
org.gradle.parallel=true
org.gradle.logging.level=quiet
model.all.version=1.0.0
```
Significantly speeds up the build. Quiet doesn't suppress the output of smithy validation, but it does clean up the output a bit. I checked to see if this still works with the update_versions script, and it seems like it does.